base/src/main/res/values/styles.xml

```diff
@@ -7,6 +7,7 @@
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
+        <item name="android:fontFamily">@font/ideal_sans_medium</item>
```

Comment:
I think we use medium fonts for title, subtitle, buttons etc where the text should be bold. IMP we should use regular font here, maybe ideal_sans_book.

Reply:
I've made the necessary change.